docs/modeling-your-data/modeling-your-data-with-dbt/dbt-advanced-usage/dbt-advanced-operation/index.md

```diff
@@ -67,8 +67,15 @@ All the incremental models in the Snowplow packages have recommended cluster key
 
 </Tabs>
 
-## Overriding Macros
+### Overriding Macros
```

Comment:
Why H3? Other titles on this page are H2, and I'm not sure this purely is a subheading of `Cluster Keys` is it (since it mentioned other macros)?